libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -32,9 +32,12 @@
 #import "SFSDKLoginHostListViewController.h"
 #import "SFSDKLoginHostDelegate.h"
 #import "UIColor+SFColors.h"
+#import "SFSDKResourceUtils.h"
+#import "SFUserAccountManager.h"
+#import "SFAuthenticationManager.h"
 
 
-@interface SFLoginViewController () <SFSDKLoginHostDelegate, SFUserAccountManagerDelegate, SFAuthenticationManagerDelegate>
```

Comment:
`SFAuthenticationManagerDelegate` wasn't actually implemented, so I took it out.